readme.md

```diff
@@ -109,6 +109,28 @@ Or you could just use plain JavaScript:
 </div>
 ```
 
+### `<For>`
+
+React component that iterates over `of` prop and renders the children.
```

Comment:
This is not correct. It doesn't render the `children`, but rather the result the `render` prop.